### PR TITLE
Add offline-external-js to empty edition

### PR DESCRIPTION
**Pull request by simonbaird**
Currently I'm building these myself for use on tiddlyhost.com. I'm thinking it would be nicer if they were built and distributed by TiddlyWiki's own build automation, so this is a step towards that. WDYT?

### CODE REVIEW

**Comment by simonbaird** [comment]
The result of this PR is the two extra files which you can confirm did show up on the the preview:
* https://tiddlywiki5-git-fork-simonbaird-add-offlineext-9c6327-jermolene.vercel.app/emptyx.html
* https://tiddlywiki5-git-fork-simonbaird-add-offlineext-9c6327-jermolene.vercel.app/tiddlywikicore-5.3.2-prerelease.js

File: editions/empty/tiddlywiki.info [hunk 1]
```diff
@@ -12,6 +12,9 @@
 		"empty": [
 			"--render","$:/core/save/all","empty.html","text/plain",
 			"--render","$:/core/save/all","empty.hta","text/plain"],
+		"emptyexternalcore": [
+			"--render","$:/core/save/offline-external-js","emptyx.html","text/plain",
```
Comment (simonbaird) [inline editions/empty/tiddlywiki.info:16]: Could make the file name longer, e.g:

```suggestion
			"--render","$:/core/save/offline-external-js","empty-external-core.html","text/plain",
```

File: editions/empty/tiddlywiki.info [hunk 2]
```diff
@@ -12,6 +12,9 @@
 		"empty": [
 			"--render","$:/core/save/all","empty.html","text/plain",
 			"--render","$:/core/save/all","empty.hta","text/plain"],
+		"emptyexternalcore": [
```
Comment (simonbaird) [inline editions/empty/tiddlywiki.info:15]: Maybe:

```suggestion
		"emptyx": [
```

File: editions/empty/tiddlywiki.info [hunk 3]
```diff
@@ -12,6 +12,9 @@
...
+			"--render","$:/core/save/offline-external-js","emptyx.html","text/plain",
+			"--render","$:/core/templates/tiddlywiki5.js","[[tiddlywikicore-]addsuffix<version>addsuffix[.js]]","text/plain"],
```
Comment (simonbaird) [inline editions/empty/tiddlywiki.info:17]: This matches the similar line in `editions/server-external-js/tiddlywiki.info`

**Comment by pmario** [comment]
>[. . .] I'm thinking it would be nicer if they were built and distributed by TiddlyWiki's own build automation, so this is a step towards that. WDYT?

I'm in favour of this PR.

**Comment by Jermolene** [comment]
Thanks @simonbaird – I think using the filename "empty-external-core.html" would indeed be clearer.

**Comment by simonbaird** [comment]
> Thanks @simonbaird – I think using the filename "empty-external-core.html" would indeed be clearer.

PR updated to use that file name.

File: editions/empty/tiddlywiki.info (hunk 1 above)
Comment (simonbaird) [reply editions/empty/tiddlywiki.info]: Done.

**Pull request merged**